f2/src/components/formik/fileUpload/index.js

```diff
@@ -18,7 +18,7 @@ export const FileUpload = ({ field, form, onChange, ...props }) => {
     <FormRow>
       <FileInput id={props.id} custom>
         <FormFile.Input accept={acceptableExtensions} onChange={onChange} />
-        <FormFile.Label />
+        <FormFile.Label>lable={props.label}</FormFile.Label>
```

Comment:
This should only include the props label. Please change from `lable={props.label}` to `{props.label}`.

Reply:
fixed. Thanks.